yarn.lock

```diff
@@ -2644,7 +2644,7 @@
     jss-plugin-vendor-prefixer "^10.8.2"
     prop-types "^15.7.2"
 
-"@mui/system@^5.6.4":
+"@mui/system@^5.2.8", "@mui/system@^5.6.4":
```

Comment:
If it's a dep, we can put it in its latest version like `@mui/utils` right ?

Reply:
I think that the wider the range we set, the better, it will minimize version duplication among developers.